core/block.js

```diff
@@ -1415,15 +1388,14 @@ Blockly.Block.newFieldImageFromJson_ = function(options) {
   var height =
     Number(Blockly.utils.replaceMessageReferences(options['height']));
   var alt = Blockly.utils.replaceMessageReferences(options['alt']);
-  var flip_rtl = Blockly.utils.replaceMessageReferences(options['flip_rtl']);
-  return new Blockly.FieldImage(src, width, height, alt, flip_rtl);
+  return new Blockly.FieldImage(src, width, height, alt);
```

Comment:
PR where it was introduced (via @rachel-fenichel ): https://github.com/LLK/scratch-blocks/commit/470d10af102319fbc1a97ef7f34f84ea599333f7